src/styles/blocks/card.scss

```
@@ -0,0 +1,118 @@
+/* #region Card */
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+
+  box-sizing: border-box;
+  width: 1024px;
+  justify-content: space-between;
+  // width: 2 * $paddingX + 3 * $gapX + 4 *$card-width;
+  // gap: 46px 48px ;
+  margin: 0 auto;
+  padding: 50px $cards-paddingX;
+
+  &__card {
+    margin-right: 48px;
+    margin-bottom: 46px;
+
+    &:nth-child(4n) {
+      margin-right: 0;
+    }
+
+    &:nth-last-child(-n+4) {
+      margin-bottom: 0;
+    }
+  }
+}
+
+.card {
+  box-sizing: border-box;
+  width: $card_width;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  border: 1px solid #e5e5e5;
+  border-radius: 5px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__tittle {
+    margin: 0;
+    margin-bottom: 4px;
+
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $primary-text-color;
+  }
+
+  &__code {
+    margin: 0;
+    margin-bottom: 16px;
+    color: $secondary-text-color;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    font-weight: 400;
```

```suggestion
    justify-content: space-between;
    align-items: center;
    font-weight: 400;
```
I believe this should fix the error you have in the card__review-text.

---

Thanks this is also done! I believe my Catalog is done properly! but for some reason I keep getting some hover test failed which I dont understad.

---

Perfect! :) Yeah don't know either. Put it on mentor review so maybe someone can help you with this.